### PR TITLE
docs: Update Draft TRG 1-4

### LINKED ISSUE #427: feat(trg-1-4): Adapt and add use of draw.io
## What

[TRG-1-4](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-1-4) describes the usage of diagram as code (like mermaid, PlantUML). Since there is sometimes the need to create more complex diagrams, we should add draw.io also to this TRG.

**Import** Its should be only allowed to provide SVG files in the static folder, cause they can be interpreted by docusaurs and also can be edited by everybody within desktop or online version of draw.io

## How

-

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description
- add reference to draw.io 
- add draw.io example

fixes #427 

ongoing IP-Validation Checks
- [x] https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11149
- [x] https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11150
- [x] https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11151
- [x] https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11152

## Preview
image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs. 
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/release/trg-0/trg-1-4.md
Comment (stephanbcbauer) [inline docs/release/trg-0/trg-1-4.md:21]: Maybe there is a misunderstanding on my side, but the documentation says the svg file must be stored in the `docs` folder, but the example svg file is stored in `static/img/alice-bob-drawio.svg`

**Review by stephanbcbauer — CHANGES_REQUESTED** [review]
@FaGru3n to be honest i am not sure if a change is needed :)

**Comment by stephanbcbauer** [comment]
> We need to clarify this TRG a little bit more:
> 
> Generally speaking i don't think its enough to just have a *.svg file available because, while you can change it (its plain text) without knowhing which tool you used to generate it, you can't really change it. You can fix existing text, you might be able to update some lines etc. but not 'change' the content.
> 
> We might even make it mandatory to have a short section in the README.md to clarify which tool they use. draw.io for example is a good option but if you need a speicifc version you might need to document how to use it with something like a container image we host ourselfes and we need to align on a version.

@Siegfriedk not sure if this helps, but at least in the SVG file which were created by draw.io there is always this note in the header
image

**Comment by FaGru3n** [comment]
> bfdc244

**Review by stephanbcbauer — APPROVED** [review]
LGTM Thank you

**Comment by FaGru3n** [comment]
I will wait for @Siegfriedk for the approval before merge 😉

File: docs/release/trg-0/trg-1-4.md
Comment (SebastianBezold) [inline docs/release/trg-0/trg-1-4.md:85]: Is the used version really important here? Are there migration issues with saved draw.io exports? 
If so, I would add a small hint, how to find out, which version of draw.io is live, when using the public version.
What is missing from my point of view. is the reference to the draw.io export. AFAIK, you need to store the XML based `.drawio` file in order to import and edit it again

File: docs/release/trg-0/trg-1-4.md
Comment (SebastianBezold) [inline docs/release/trg-0/trg-1-4.md:19]: "code languages" -> maybe diagram as code tool? never heard the term "code languages" before
languages, or -> comma missing
If `you're` instead of `your`.
What is completely missing from my point of view is a specific example on how to document in the `README.`
Maybe this is not as relevant for product repos as it will be for the website. But especially for the website, we have a lot of different contributors and very deep directory structure. If we clutter our top level `README` with various instruction on how to handle a file deep into our directory structure, that will potentially mess up our entrypoint to the repo. 
Have there been thoughts on how to handle this in our website repo already?

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
I think if we want to restrict contributors to specific tools, then we need to provide more guidance. 
Alternative would be only mention the tools as valid options (golden path) and only have mandatory guidelines, like:

- You need to contribute the "raw" editable source
- Used tools need to be usable by anyone without the need of a licence or similar (prefer open source tools)
- Tools should be OS independent

and so on

File: docs/release/trg-0/trg-1-4.md
Comment (FaGru3n) [reply docs/release/trg-0/trg-1-4.md]: fixed typos and update the current one to just document it, not exactly where. 

[chore: restructure content + update example diagram](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/429/commits/f3adad9638f7fbfec8c422bd4f82d6ddc0edb740)

**Comment by FaGru3n** [comment]
updated my branch current view:

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**
